profitbricks.go

```diff
@@ -50,6 +50,13 @@ func RestyClient(username, password, token string) *Client {
 	})
 	c.AddRetryCondition(
 		func(r *resty.Response, err error) bool {
+			// Only retry on read-only requests
+			if r.Request.Method != http.MethodGet &&
```

Comment:
The switch-case construct below looks better ;)

Reply:
Well, I did not want to have any retries for these methods, so i figured it would be clearer to opt out fast.

Reply:
Yes, your fast opt out was totally fine, I just wanted to say that your longish `if` condition looks bad compared to the beautiful `switch` right below.